### PR TITLE
fix lubridate operation for subtraction.

### PR DESCRIPTION
**Pull request by hidekoji**
# Description

This is to address the below` lubridate` operation issue.

## Test
```r
## years with %m-%
>as.Date("2020-02-29") %m-% lubridate::period(2, units = "years")
[1] "2018-02-28"

## years with - (NG case for the reord)
> as.Date("2020-02-29") - lubridate::period(2, units = "years")
[1] NA

## months with -  (NG case for the record)
> as.Date("2020-03-30") - lubridate::period(1, units = "months")
 [1] NA

## months with %m-% it
> as.Date("2020-03-30") %m-% lubridate::period(1, units = "months")
[1] "2020-02-29"

## weeks with -
as.Date("2020-02-29") - lubridate::period(4, units = "weeks")
"2020-02-01"

## days with -
as.Date("2020-02-29") - lubridate::period(29, units = "days")
[1] "2020-01-31"

## hours with -
> as.Date("2020-02-29") - lubridate::period(24*30, units = "hours")
[1] "2020-01-30"

### minutes with -
as.Date("2020-02-29") - lubridate::period(24*31*60, units = "minutes")
[1] "2020-01-29"

### seconds with -
> as.Date("2020-02-29") - lubridate::period(24*28*60*60, units = "seconds")
[1] "2020-02-01"
```

# Checklist
Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [ ] Add test cases for this fix/enhancement
- [x] Pass devtools::check()
- [x] Pass devtools::test()
- [ ] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

File: R/google_analytics.R [hunk 1]
```diff
@@ -77,15 +77,15 @@ getGoogleAnalytics <- function(tableId, lastNDays = 30, dimensions, metrics, tok
...
     endDate = as.character(lubridate::today() - 1);
   } else if (dateRangeType == "lastYearToYesterday") {
-    startDate = as.character(lubridate::floor_date(lubridate::today(), "year") - lubridate::years(1));
+    startDate = as.character(lubridate::floor_date(lubridate::today(), "year") %m-% lubridate::years(1));
     endDate = as.character(lubridate::today() - 1);
   }
 
   if(is.null(startDate)) {
-    startDate <- as.character(lubridate::today() - lubridate::days(lastNDays))
+    startDate <- as.character(lubridate::today() %m-% lubridate::days(lastNDays))
```
Comment (hideaki) [inline R/google_analytics.R]: For days we can use -.

File: R/get_stripe_data.R [hunk 2]
```diff
@@ -46,7 +46,7 @@ get_stripe_data <- function(endpoint = "balance/history",
       if(!date_type %in% c("days", "weeks", "months", "years")){
         stop("date_type must be \"days\", \"weeks\", \"months\", \"years\" or \"exact\"")
       }
-      date_since <- lubridate::today() - lubridate::period(as.numeric(date_since), units = date_type)
+      date_since <- lubridate::today() %m-% lubridate::period(as.numeric(date_since), units = date_type)
```
Comment (hideaki) [inline R/get_stripe_data.R]: We should do this only for month and year.

File: R/get_mailchimp_data.R [hunk 3]
```diff
@@ -33,7 +33,7 @@ get_mailchimp_data <- function(endpoint = "export/1.0/list", date_type = "exact"
       if(!date_type %in% c("days", "weeks", "months", "years")){
         stop("date_type must be \"days\", \"weeks\", \"months\", \"years\" or \"exact\"")
       }
-      date_since <- lubridate::today() - lubridate::period(as.numeric(date_since), units = date_type)
+      date_since <- lubridate::today() %m-% lubridate::period(as.numeric(date_since), units = date_type)
```
Comment (hideaki) [inline R/get_mailchimp_data.R]: We should do this only for month and year.

File: R/google_analytics.R (hunk 1 above)
Comment (hidekoji) [reply R/google_analytics.R]: thank you for catching it. changed it to use `-`

File: R/get_stripe_data.R (hunk 2 above)
Comment (hidekoji) [reply R/get_stripe_data.R]: Updated per review.

File: R/get_mailchimp_data.R (hunk 3 above)
Comment (hidekoji) [reply R/get_mailchimp_data.R]: Updated per review.

**Pull request merged**